MPower/app/src/main/assets/json/consent_section.json

```diff
@@ -59,135 +59,134 @@
     "incorrectIcon": "rss_ic_quiz_retry",
     "correctIcon": "rss_ic_quiz_valid"
   },
-  "sections":
-  [
+  "sections": [
     {
-      "sectionType" : "overview",
-      "sectionTitle" : "Welcome",
-      "sectionSummary" : "This overview explains the research study. After you learn about the study, you can choose if you would like to participate. This may take about 20 minutes to complete.",
-      "sectionLearnMoreButtonTitle" : "Learn more",
-      "sectionHtmlContent" : "consent_1welcome"
+      "sectionType": "overview",
```

Comment:
It'd be ideal if the json and html were synced, maybe versioned in a separate repo and included via git submodule or git subtree